.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-#Requires review from DevOps when making changes to the following files/folders
+#Requires below review when making changes to the following files/folders
```

Comment:
`Changes to the following files/folders require review from listed teams` might read better